app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :email, presence: true
```

Comment:
Maybe you can add additional validation for uniqueness. **[OPTIONAL]**
I can currently create multiple users with the same email and username. see image below
![Screen Shot 2019-09-06 at 8 57 00 AM](https://user-images.githubusercontent.com/31193823/64441274-8d430a00-d09b-11e9-9804-8af4a1c28982.png)